blazor/dropdown-list/style.md

```diff
@@ -270,6 +282,64 @@ In the following code, a single list Item is hidden using jsinterop.
 
 ![Blazor DropDownList with Popup Width](./images/style/blazor_dropdown_disable-listitem.png)
 
+## Adding conditional HTML attribute to list item
+
+You can achieve adding attributes to the li items based on datasource value with the help of JSInterop. In the, Opened event need to call the client side script by passing the required arguments (datasource and id) and add the attributes based on the datasource value obtained from the server.
+
+% highlight cshtml %}
+
+{% include_relative code-snippet/style/add-attribute-listitem.razor %}
+
+{% endhighlight %}
+
+{% tabs %}
+{% highlight razor tabtitle="~/_Layout.cshtml" %}
+
+    <script> 
+        function OnCreated(datasource, id) { 
+            setTimeout(() => { 
+                //Here popup element is uniquely identified with id. 
+                //Classes added via CssClass property will be added to the popup element also. 
+                //You can also uniquely identify the popup element with the help of added class. 
```

Comment:
//You can also uniquely identify the popup element with the help of the added classes.